test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/LoadTest.java

```diff
@@ -43,8 +44,14 @@
 	public static OptionalInt testDurationMinutes = OptionalInt.empty();
 	public static OptionalInt threadNumber = OptionalInt.empty();
 	public static OptionalInt hcsSubmitMessage = OptionalInt.empty();
+	public static OptionalInt hcsSubmitMessageSizeVar = OptionalInt.empty();
 	/** initial balance of payer account used for paying for performance test transactions */
 	public static OptionalLong initialBalance = OptionalLong.of(900_000_000_000L);
+	public static OptionalInt totalTestAccounts = OptionalInt.empty();
+	public static OptionalInt totalTestTopics = OptionalInt.empty();
```

Comment:
Please provide comments for why these newly added properties can be used

Reply:
Added in PerfTestLoadSettings class.